eth/mgr/mgr.go

```diff
@@ -0,0 +1,136 @@
+package mgr
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/ledgerwatch/turbo-geth/common"
+	"github.com/ledgerwatch/turbo-geth/ethdb"
+	"github.com/ledgerwatch/turbo-geth/trie"
+)
+
+const (
+	TicksPerCycle  uint64 = 4096
+	BlocksPerTick  uint64 = 50
+	BlocksPerCycle uint64 = BlocksPerTick * TicksPerCycle
```

Comment:
If we set BlocksPerTick 50, then we should set TicksPercycle to something like 100 :) Otherwise one cycle is going to be like 2 months